cfgov/tccp/jinja2/tccp/includes/speed_bump.html

```diff
@@ -1,18 +1,12 @@
 {%- macro speed_bump(value) -%}
```

Comment:
Can we add a second argument to `speed_bump`, like `compact` or some similar switch that can toggle a class? We're reusing the speed bump on the card details page, where it doesn't need to be as prominent:

![speed-bump](https://github.com/cfpb/consumerfinance.gov/assets/1862695/5f8a1893-027f-41e4-af60-29657a646f7d)

That's just a few small style tweaks to what you have in this PR (20px padding, making the paragraphs inline, and bolding the first paragraph). So maybe we could do something like…

```jinja
<article class="o-well o-well{{ '--compact-speed-bump' if compact else '--speed-bump' }}">
```

…and then add the appropriate styles. It doesn't have to be a `compact` argument if something else makes more sense, but some sort of selector so we can use the same macro on both the list page and details page would be amazing.

Reply:
That works for me! I'll push up a change.

Reply:
@niqjohnson with https://github.com/cfpb/consumerfinance.gov/pull/8334/commits/c4b19b6274fd3caf0b974defeb5de7a5b6f6394e I added a flag to make them speed bump "large" when desired. Here's what it looks like on the card results page vs the card detail page:

| `large` | not `large` |
|--------|-------|
| <img width="1195" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/1060248/29eda53b-10ae-4367-b1c0-2b159fb5921e"> | <img width="821" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/1060248/227554c4-cbc1-4f15-9447-219cd5744d3f"> |